### PR TITLE
[SAT-63] Configure logging to a file.

### PR DESCRIPTION
**Pull request by rozenbom**
## Description of changes in release / Impact of release:
Configure logging to a file.

### Is this a breaking change?
  - [ ] Yes
  - [x] No

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)

### CODE REVIEW

**Comment by aslepakurov** [comment]
All good, but when running `make check` getting
```
======================================================= short test summary info =======================================================
FAILED satellite/tests/proxy/test_process.py::test_start_stop - satellite.proxy.exceptions.ProxyError: Unable to start regular proxy.
============================================= 1 failed, 48 passed, 1 deselected in 2.32s ==============================================
```

**Comment by aslepakurov** [comment]
Other then that 👍

**Comment by rozenbom** [comment]
> All good, but when running `make check` getting
> 
> ```
> ======================================================= short test summary info =======================================================
> FAILED satellite/tests/proxy/test_process.py::test_start_stop - satellite.proxy.exceptions.ProxyError: Unable to start regular proxy.
> ============================================= 1 failed, 48 passed, 1 deselected in 2.32s ==============================================
> ```

Are you sure that there were no running proxies on your machine when you run this check?

**Review by aslepakurov — COMMENTED** [review]
Add timestamp to log, pls

File: Makefile [hunk 1]
```diff
@@ -14,3 +14,7 @@ test_dist:
 
 clean:
 	find satellite -name "*.pyc" | xargs -I {} rm -rf {}
+
+# Run this before push
+check: lint test dist test_dist
```
Comment (aslepakurov) [inline Makefile:19]: Yeap, still getting error here.
- Satellite not running
- Ports 9098/9099 not open
- no config file

**Comment by rozenbom** [comment]
> Add timestamp to log, pls

done

**Review by aslepakurov — APPROVED** [review]
LGTM

**Pull request merged**